src/fairness/weighted_tree/test/load_accounting_db.cpp

```diff
@@ -112,50 +156,42 @@ std::shared_ptr<weighted_tree_node_t> get_sub_banks (
                             sqlite3 *DB,
                             const std::string &bank_name,
                             std::shared_ptr<weighted_tree_node_t> parent_bank,
-                            sqlite3_stmt *b_select_shares_stmt,
-                            sqlite3_stmt *b_select_sub_banks_stmt,
-                            sqlite3_stmt *b_select_associations_stmt)
+                            sqlite3_stmt *c_shrs,
```

Comment:
I like this function is now returning the root of the subtree, not the tree. We should revise the comment accordingly. 

```diff
> It returns a shared pointer to the root of the tree.
< It returns a shared pointer to the root of the subtree.
```